modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/index/IndexSynchronizer.java

```diff
@@ -24,8 +24,6 @@ public interface IndexSynchronizer {
 	public void synchronizeIndexDefinition(
 		IndexDefinitionData indexDefinitionData);
 
-	public void synchronizeIndexes();
```

Comment:
@W2jx I do not understand why a merge needs to remove an API method. Please separate your changes into two commits.